include/pka_config.h

```diff
@@ -78,8 +75,8 @@
 // Macro for mapping PKA Ring address into Window RAM address. It converts the
 // ring address, either physical address or virtual address, to valid address
 // into the Window RAM. This is done assuming the Window RAM base and size.
-#define PKA_RING_MEM_ADDR(addr, size) \
-    (PKA_WINDOW_RAM_BASE | (((addr) & 0xffff) | \
+#define PKA_RING_MEM_ADDR(base, addr, size) \

```

Comment:
I think this macro should return the offset within the window ram, as per the new definition. right?

Reply:
I will discuss this offline. I'm somewhat confused with this macro. We were always using PKA_WINDOW_RAM_BASE before but what if SPLIT_MODE is enabled ? will this macro still do the mapping correctly ?